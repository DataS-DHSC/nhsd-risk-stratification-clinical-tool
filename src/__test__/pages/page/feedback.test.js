import React from 'react';
import { render } from '@testing-library/react';
import Feedback from '../../../pages/ras/page/feedback';

const setup = () => render(<Feedback />);

describe('feedback should match snapshot', () => {
  it('should match snapshot', () => {
    const container = setup();
    expect(container.asFragment()).toMatchSnapshot();
  });
});
