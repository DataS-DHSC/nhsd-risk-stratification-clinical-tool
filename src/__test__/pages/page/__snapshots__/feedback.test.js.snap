// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`feedback should match snapshot should match snapshot 1`] = `
<DocumentFragment>
  <h1>
    Loading survey
  </h1>
</DocumentFragment>
`;
